docs/user-guide/persistent-volumes/index.md

```diff
@@ -29,11 +44,21 @@ PVs are resources in the cluster.  PVCs are requests for those resources and als
 
 ### Provisioning
 
-A cluster administrator will create a number of PVs. They carry the details of the real storage which is available for use by cluster users.  They exist in the Kubernetes API and are available for consumption.
+There are two ways PVs may be provisioned: statically or dynamically.
+
+#### Static
+A cluster administrator creates a number of PVs. They carry the details of the real storage which is available for use by cluster users.  They exist in the Kubernetes API and are available for consumption.
+
+#### Dynamic
+A cluster administrator creates a number of `StorageClasses`. They describe "classes" of PVs that cluster users may request. They contain the information that the cluster needs in order to dynamically create PVs in response to user requests.
```

Comment:
this way it looks that StorageClass is for dynamic provisioning only. IMO, we could just mention here that when no matching "static" PV is found, Kubernetes will try to dynamically provision one and this provisioning is based on classes.

Classes themselves could be described somewhere else.